File: hash.cpp
#include <iostream>
#include <map>

int main()
{
    std::map<std::string, std::string> dict;
    dict["brand"] = "ford";
    dict["model"] = "Mustang";
    dict["year"] = "1964";

    std::string x = "brand";

    // Check if x is a key in the dictionary
    bool isKeyPresent = (dict.find(x) != dict.end());

    // Print the result
    std::cout << std::boolalpha << isKeyPresent << std::endl;

    return 0;
}

File: HashMap/Hash.java
import java.util.HashMap;
public class Hash {

    Hash(){


    }
    public static void main(String[] args) {
    HashMap<String, String> capitalCities;
    HashMap<String, String> dict;
    capitalCities = new HashMap<>();
    dict = new HashMap<>();
    dict.put("brand", "ford");
    dict.put("model", "mustang");
    dict.put("year", "1964");
    capitalCities.put("England","London");
    capitalCities.put("Germany","Berlin");
    capitalCities.put("Norway","Oslo");
    System.out.println(dict);
    System.out.println(capitalCities);
    }
}
